lib/client.js

```diff
@@ -15,18 +15,111 @@ var brMessages = require('bedrock-messages');
 var request = require('request');
 var scheduler = require('bedrock-jobs');
 var uuid = require('node-uuid').v4;
+require('bedrock-express');
 
 require('./config');
 
+var api = {};
+module.exports = api;
+
+// TODO: Add logging info
```

Comment:
I think I was logging some debugging info but then I got rid of it, the TODO is there to remind me to log some relevant info later to help with debugging.